metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -41,7 +42,27 @@ event. In the above example, this will cause the physical_disk.write.per_sec
 and physical_disk.write.time.pct measurements to be sent as a single event.
 The default behaviour is for all measurements to be sent as separate events.
 
-*`counters`*:: Counters specifies a list of queries to perform. Each individual
+*`append_instance_counter`*:: A boolean option to retrieve the full instance name with the included
+ counter, ex.
+----
+ "perfmon" : {
```

Comment:
Indentation looks off here and below. Not sure if it's just Github (did not build the docs).